oci/layer/unpack.go

```diff
@@ -149,18 +149,19 @@ func UnpackManifest(ctx context.Context, engine cas.Engine, bundle string, manif
 
 // UnpackRootfs extracts all of the layers in the given manifest.
 // Some verification is done during image extraction.
-func UnpackRootfs(ctx context.Context, engine cas.Engine, rootfsPath string, manifest ispec.Manifest, opt *MapOptions) (err error) {
+func UnpackRootfs(ctx context.Context, engine cas.Engine, rootfsPath string, manifest ispec.Manifest, opt *MapOptions, allowExistingRoot bool) (err error) {
```

Comment:
I'd prefer if this wasn't done through a boolean flag. Since you're the only user of `UnpackRootfs` I'm fine with just always allowing unpacking into an existing rootfs, and inside `UnpackManifest` we can do the `os.Lstat(...) == nil` check because there we are operating with bundles.

Reply:
fixed, thanks.